manifests/manager/manager.yaml

```diff
@@ -15,8 +15,6 @@ spec:
       labels:
```

Comment:
Do we need [Controller Manager Config](https://github.com/kubeflow/tf-operator/blob/e358768661156c3ff1b8b22126090f6dae2374b8/manifests/manager/controller_manager_config.yaml) ? Are we going to use it somewhere ?

Reply:
For leader election, We do plan to use this way. I ask Deepak to help create an issue to track it. In this PR, we can delete it.

Reply:
I think, that would be better.
In the further PRs we can decide the location for this Config.

Reply:
once we enable leader election, we would want to use this config https://github.com/kubeflow/tf-operator/issues/1350